src/types.rs

```diff
@@ -122,6 +122,22 @@ pub struct HostStats {
     pub timestamp: Option<i64>,
 }
 
+// Type for the list/available endpoint
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(crate = "rocket::serde")]
+#[serde(rename_all = "camelCase")]
+pub struct HostLatest {
+    #[serde(rename = "_id")]
+    pub _id: String,
```

Comment:
```suggestion
    #[serde(rename = "holoportId")]
    pub _id: String,
```
I understand why we need "_id" in the mongodb env, but couldn't we just rename it after de-serializing it on this end of the api?